Source/Chatbook/UI.wl

```diff
@@ -573,6 +675,17 @@ makeFrontEndAndNotebookSettingsContent[
 	]
 ]
 
+(*======================================*)
+
+showAdvancedModelsQ[] :=
+	TrueQ @ CurrentValue[$FrontEnd, {
+		PrivateFrontEndOptions,
```

Comment:
I think we should be using `TaggingRules` and the scope where the menu is being displayed, because this will need to work in cloud as well. Assigning values to `$FrontEnd` won't work there.

<img width="558" alt="image" src="https://github.com/WolframResearch/Chatbook/assets/6674723/d083f046-5145-4fdb-985c-c5299ed0ea65">

Reply:
This option is only available at the global scope, it's part of the main preferences menu. So setting it on a NotebookObject I think doesn't apply.

What is the best way to store "global" state in the cloud?

Additionally, at the moment, in the Cloud the snapshot models won't ever appear, which I reasoned was a decent tradeoff to keep the cloud docked cell toolbar interface simple.